ros2_modulo_control/Dockerfile

```diff
@@ -17,6 +16,26 @@ RUN git clone -b galactic --depth 1 https://github.com/ros2/rcl_interfaces.git
 RUN git clone -b galactic --depth 1 https://github.com/ros2/test_interface_files.git
 RUN git clone -b ros2 --depth 1 https://github.com/ros/angles.git
 
+
+FROM ${BASE_IMAGE}:humble as ros2-control-version-humble
```

Comment:
Same issue here, though we could stick with only `humble-devel` for now. Ultimately I think the same approach should be taken as with galactic / galactic-devel